config/migrations/2025/old-consumer-flush/20250220163123-remove-identifiers.sparql

```diff
@@ -0,0 +1,7 @@
+PREFIX dct: <http://purl.org/dc/terms/>
+
+DELETE WHERE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?s dct:identifier ?o .
+  }
+}
```

Comment:
This should be gone with `20250220161209-remove-triples-public-graph.sparql` no ?